deluge/ui/gtk3/filtertreeview.py

```diff
@@ -89,10 +89,13 @@ def __init__(self):
         self.treeview.set_headers_visible(False)
         self.treeview.set_level_indentation(-21)
         # Force theme to use expander-size so we don't cut out entries due to indentation hack.
-        Gtk.rc_parse_string(
-            """style "treeview-style" {GtkTreeView::expander-size = 7}
-                            class "GtkTreeView" style "treeview-style" """
-        )
+        style_provider = Gtk.CssProvider()
+        css = b""" * { -GtkTreeView-expander-size: 14;} """
+        try:
+            style_provider.load_from_data(css)
+        except Exception as ex:
```

Comment:
Please don't use a bare a Exception, did you encounter any errors that require the try..except? From the docs it would raise a Glib.Error. Also no need for `pass` and print statement should be log.error